### PR TITLE
Update bal_tools version to get proper subgraphs.

### PR DESCRIPTION
**Pull request by Tritium-VLK**

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Test run here: https://github.com/BalancerMaxis/protocol_fee_allocator/actions/runs/10082783348/job/27877820666

Based on the current setup if it succeeds it will push changes to the branch used even though it is not run from main.

**Review by jalbrekt85 — APPROVED** [review]
changes will start working with merge of: https://github.com/BalancerMaxis/bal_addresses/pull/386

File: requirements.txt [hunk 1]
```diff
@@ -4,5 +4,4 @@ gql[requests]==3.4.1
 pycoingecko==3.1.0
 pandas>2.0,<2.3
 simplejson==3.19.2
-git+https://github.com/BalancerMaxis/bal_addresses@0.9.7
-git+https://github.com/BalancerMaxis/bal_tools@v0.0.6
```
Comment (jalbrekt85) [inline requirements.txt:8]: we still need this bal_tools dep, right? just needs to bumped to v0.1.1

Comment (Tritium-VLK) [reply requirements.txt:8]: It's easier to not include it and just let it come with bal_addresses.  Will be easier once we get to pypi, should do that soon.

**Comment by Tritium-VLK** [comment]
Run: https://github.com/BalancerMaxis/protocol_fee_allocator/actions/runs/10092661221

**Comment by Tritium-VLK** [comment]
Run: https://github.com/BalancerMaxis/protocol_fee_allocator/actions/runs/10092661221/job/27909037608#step:2:54

**Pull request merged**